.github/workflows/ci.yml

```
@@ -0,0 +1,227 @@
+on:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main, develop ]
+
+env:
+env:
+  GO_VERSION: "1.23"
+
+jobs:
+  test:
```

actionlint (1.7.7)

"steps" section is missing in job "test"

(syntax-check)